HabitTracker.BatataDoc3/db/CRUD.cs

```diff
@@ -0,0 +1,159 @@
+﻿using Microsoft.Data.Sqlite;
+
+
+namespace HabitTracker.BatataDoc3.db
+{
+
+    internal class Crud
+    {
+        private SqliteConnection? conn;
+
+        public Crud()
+        {
+            conn = null;
+        }
+
+        public bool StartDatabase()
+        {
+            String sql = @"CREATE TABLE habits(
+                id INTEGER PRIMARY KEY AUTOINCREMENT,
+                name TEXT,
+                measure TEXT,
+                quantity INT,
+                date DATE
+            )";
+
+            bool exists = CheckIfDbExists();
+            try 
+            {
+                conn = new SqliteConnection(@"Data Source=db\habits.db");
```

Comment:
This also needs to be a using declaration to stop memory leaks, collisions on an existing open connection:
`using var connection = new SqliteConnection(connectionString);`

I would get away from having a global connection value and passing that around. Only connect to the database when you need to.